Containers/mastercontainer/start.sh

```diff
@@ -15,9 +15,20 @@ elif ! mountpoint -q "/mnt/docker-aio-config"; then
     exit 1
 elif ! sudo -u www-data test -r /var/run/docker.sock; then
     echo "Trying to fix docker.sock permissions internally..."
-    GROUP="$(stat -c '%g' /var/run/docker.sock)"
-    groupadd -g "$GROUP" docker && \
-    usermod -aG docker www-data
+
+    DOCKER_GROUP=$(stat -c '%G' /var/run/docker.sock)
+    DOCKER_GROUP_ID=$(stat -c '%g' /var/run/docker.sock)
+    # check if a group with the same group id of /var/run/docker.socket already exists in the container
+    if grep -q $DOCKER_GROUP_ID /etc/group; then
```

Comment:
Thanks for the refactoring! For fail-proof I'd recommend to use the following code:
```suggestion
    if grep -q "^$DOCKER_GROUP:" /etc/group; then
```